### ISSUE TITLE
v7: Restrict use of the term "node" to version 6

### ISSUE THREAD

**Issue by broofa** [uuid6/uuid6-ietf-draft#49]
`node` was originally defined as "MAC address or a random value".  Then in this spec we've decided to be explicit about the fact it should not be a MAC address.  Thus, the implication is that `node` MUST be a random value.

Meanwhile the definition of the `subsec_seq_node` field makes no mention of "node", other than in the name:

> The remaining 62 bits which MAY be allocated to any combination of additional sub-second precision, sequence counter, or pseudo-random data.

(But it's at least explicit about the field holding subsecond time, sequence, or **random** data)

However in section 4.4.3 we define "node" as "the leftover bits" that aren't allocated to subsecond or sequence values, then rather tepidly state, "In most scenarios the node SHOULD be filled with pseudo-random data".

But that opens the door for it being something other than random values.  But no mention is made of what else might be used there, or how that would be possible without falling afoul of the implied (legacy) definition.

Basically, its use is confusing and adds little value.  We should just be explicit about the fact the "leftover bits" should be set to pseudo-random values and avoid the term altogether unless we're talking about the legacy definition (in version 6).

**Comment by kyzer-davis** [comment]
Agreed, the only one that will reference `node` in Draft 03 will be v6 just to keep that layout parity with v1.
